src/app.py

```diff
@@ -20,37 +21,58 @@ def setup_logging(logger_name):
     logging.basicConfig(level=logging.DEBUG)
 
 
-def main(generated_files_folder='/usr/src/tmp',
+@click.command()
+@click.option('--vcf', is_flag=True, help='generates VCF files')
+@click.option('--ortho', is_flag=True, help='generates orthology files')
+@click.option('--daf', is_flag=True, help='generates DAF files')
+@click.option('--expr', is_flag=True, help='generates expression files')
+def main(vcf, ortho, daf, expr,
+         generated_files_folder=os.path.abspath(os.path.join(os.getcwd(), os.pardir)) + '/output',
          fasta_sequences_folder='sequences',
          skip_chromosomes={'Unmapped_Scaffold_8_D1580_D1567'}):
-    #generate_vcf_files(generated_files_folder, fasta_sequences_folder, skip_chromosomes)
-    #generate_orthology_file(generated_files_folder, alliance_db_version)
-    #generate_daf_file(generated_files_folder, alliance_db_version)
-    generate_expression_file(generated_files_folder, alliance_db_version)
+
+    print('INFO\t\tFiles output: ' + generated_files_folder)
+
+    if vcf is True:
+        print('INFO\t\tGenerating VCF files')
+        generate_vcf_files(generated_files_folder, fasta_sequences_folder, skip_chromosomes)
+    if ortho is True:
+        print('INFO\t\tGenerating Orthology files')
+        generate_orthology_file(generated_files_folder, alliance_db_version)
+    if daf is True:
+        print('INFO\t\tGenerating DAF files')
```

Comment:
for this sort of statement, you should use logger.info("your info here)

Reply:
The idea was to have some output to the screen to indicate what is being generated at the time. I was going to update the logger after more enhancements.